cmd/catchpointdump/database.go

```diff
@@ -70,13 +78,56 @@ var databaseCmd = &cobra.Command{
 		if err != nil {
 			reportErrorf("Unable to print key value store : %v", err)
 		}
-		err = printStateProofVerificationContext("./ledger.tracker.sqlite", outFile)
+		// state proof verification can be found on tracker db version >= 10 or
+		// catchpoint file version >= 7 (i.e staging tables)
+		if !ledgerTrackerStaging && version < 10 || ledgerTrackerStaging && version < ledger.CatchpointFileVersionV7 {
```

Comment:
I do not think version can be compared with db schema version because the `version` is catchpoint file version
```
err = cw.WriteCatchpointStateUint64(ctx, store.CatchpointStateCatchupVersion, fileHeader.Version)
```
I think just `version < ledger.CatchpointFileVersionV7` is enough

Reply:
If `ledgerTrackerStaging == false` ,`version` would represent the value of the tracker db schema version. In this case it makes sense to compare it with the tracker db schema. Am I missing something?

Reply:
it looks weird when the same var has different semantic (db schema and catchpoint file version) but I agree the logic is correct.